index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
+  var lines = changelog.split('\n');
+  var isOneLine = (lines.length === 1);
+  var deletions = 0;
+  var containsErrorMessage = false;
+  lines.filter(function (line) {
```

Comment:
I think `forEach` is what you're looking for?

It might be cleaner to filter this and then let `deletions = lines.filter(...).length`.